phpunit.xml.dist

```diff
@@ -1,11 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<!--
-DO NOT EDIT THIS FILE!
-
-It's auto-generated by sonata-project/dev-kit package.
--->
-
```

Comment:
On this repository, this file is not edited by dev-kit (https://github.com/sonata-project/dev-kit/blob/master/config/projects.yaml#L199). It is better to remove this line, otherwise it could lead to a confusion